### ISSUE TITLE
Add Setting for cache size

### ISSUE THREAD

**Issue by Mrcubix** [tachiyomiorg/tachiyomi#10262]
### Describe your suggested feature

Right now the maximum cache size is set to around 100 MB, meaning that after reaching that limit, tachiyomi encounters cache related loads, that may take 20 to multiple multiple minutes or simply a timeout on lower end devices.

100 MB is low, that limit can be reached within only a couple of chapters (5 being the lowest i have seen),
One way to lower the amount of time the user has to clean the cache when using a lower-end device would just be to increase it.
It was done after #2091 , but it still isn't enough.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.7](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.
